winlogbeat/eventlog/wineventlog.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/pkg/errors"
 	"golang.org/x/sys/windows"
 
+	"github.com/elastic/beats/winlogbeat/checkpoint"
```

Comment:
Group this import with the other elastic imports.

Reply:
done